module-2/backend/week_one.md

```diff
@@ -5,30 +5,58 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+get, puts, delete, post, patch
 2. What is Sinatra?
+sinatra is the framework we use to create web apps
 4. What is MVC?
+model view controller
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+allows order to be maintained, and read-ability to outside developers
 6. What types of variables are accessible in our view templates without explicitly passing them?
+instance vars
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
+    @name = "mr. ed"

```

Comment:
This is another instance variable. In order to use a local, we need to use slightly different syntax.

```ruby
get '/horses' do
  @count = 1
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end
```

This would give us access to the local variable `name` in the view. If we wanted to, we could also skip the assignment and still have access to the local variable `name`:

```ruby
get '/horses' do
  @count = 1
  erb :index, locals: {name: "Mr. Ed"}
end
```